engines/winnative/payload-schema.yml

```diff
@@ -0,0 +1,57 @@
+id: "http://schemas.taskcluster.net/generic-worker/v1/payload.json#"
+$schema: "http://json-schema.org/draft-04/schema#"
+title: "Payload"
+description: "This schema defines the structure of the `payload` property referred to in a Task Cluster Task definition."
+type: "object"
+required: 
+  - "command"
+  - "maxRunTime"
+properties: 
+  command: 
+    title: "Commands to run"
+    type: "array"
+    minItems: 1
+    items: 
+      type: "array"
+      minItems: 1
+      items: 
+        type: "string"
+    description: "One array per command (each command is an array of arguments). Several arrays for several commands."
```

Comment:
I think we should restrict it to one command in total. I know it sucks, but it provides better consistency across platforms...

Also we don't want people uploading scripts as part of the task-definition. I would rather focus on enabling preloaded caches...